packages/node/src/plugins/StreamingTargetPlugin.ts

```diff
@@ -1,284 +1,62 @@
-        webpack.RuntimeGlobals.shareScopeMap
-      } : global.__webpack_share_scopes__
-      var name = ${JSON.stringify(name)}
-      `;
-      const template = `(remotesConfig) => new Promise((res) => {
-      console.log('in template promise',JSON.stringify(remotesConfig))
-        executeLoad(remotesConfig).then((remote) => {
-
-          return Promise.resolve(remote.init(shareScope.default)).then(() => {
-            return remote
-          })
-        })
-          .then(function (remote) {
-            const proxy = {
-              get: remote.get,
-              chunkMap: remote.chunkMap,
-              path: remotesConfig.toString(),
-              init: (arg) => {
-                try {
-                  return remote.init(shareScope.default)
-                } catch (e) {
-                  console.log('remote container already initialized')
-                }
-              }
-            }
-            if (remote.fake) {
-              res(proxy);
-              return null
-            }
-
-            Object.assign(global.loadedRemotes, {
-              [name]: proxy
-            });
-
-            res(global.loadedRemotes[name])
-          })
-
-
-      })`;
-
-      acc.runtime[name] = `()=> ${middleware}.then((remoteConfig)=>{
-    console.log('remoteConfig runtime',remoteConfig);
-    if(!global.REMOTE_CONFIG) {
-        global.REMOTE_CONFIG = {};
-    }
-    global.REMOTE_CONFIG[${JSON.stringify(name)}] = remoteConfig;
-    ${templateStart}
-    const loadTemplate = ${template};
-    return loadTemplate(remoteConfig)
-    })`;
-
-      acc.buildTime[name] = `promise ${middleware}.then((remoteConfig)=>{
-            if(!global.REMOTE_CONFIG) {
-        global.REMOTE_CONFIG = {};
-    }
-    console.log('remoteConfig buildtime',remoteConfig);
-    global.REMOTE_CONFIG[${JSON.stringify(name)}] = remoteConfig;
-    ${templateStart};
-    const loadTemplate = ${template};
-    return loadTemplate(remoteConfig)
-    })`;
-
-      acc.hot[name] = `()=> ${middleware}`;
-
-      return acc;
-    },
-    { runtime: {}, buildTime: {}, hot: {} } as {
-      runtime: EmptyObject;
-      buildTime: EmptyObject;
-      hot: EmptyObject;
-    }
-  );
-}
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface NodeFederationContext {}
 
 class StreamingTargetPlugin {
-  private options: ModuleFederationPluginOptions;
-  private context: Context;
-  private experiments: StreamingFederationOptions['experiments'];
+  private options: NodeFederationOptions;
+  private context: NodeFederationContext;
 
-  constructor(
-    { experiments, ...options }: StreamingFederationOptions,
-    context: Context
-  ) {
-    this.options = options || ({} as ModuleFederationPluginOptions);
-    this.context = context || ({} as Context);
-    this.experiments = experiments || {};
+  constructor(options: NodeFederationOptions, context: NodeFederationContext) {
+    this.options = options || {};
+    this.context = context || {};
```

Comment:
Perhaps we can remove the `context` since we are not using it anywhere in this file.

Reply:
done